CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.0.5] - 2020-07-13
+### Added
+- ADD POETRY
```

Comment:
```suggestion
- Packaging and dependency management tool to Poetry [#207](https://github.com/scanapi/scanapi/pull/207)
```

For changelog, we use the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) pattern. Also, we add the links to the PR as a quick reference. Take a look at how it is rendered:

https://github.com/scanapi/scanapi/blob/master/CHANGELOG.md